manual/program/reviewer_guidelines.rst

```diff
@@ -0,0 +1,167 @@
+Proposal Reviewer Guidelines
+============================
+
+.. image:: reviewers.jpeg
+  :width: 400
+  :alt: me: I should really go to bed now... also me: I will just review one more SciPy proposal
+
+The Program Committee's job is to select the talks that maximize the benefit to the entire conference audience, within the space and time constraints given. Reviewers play an integral role in realizing this goal.
+
+Each reviewer has approximately 5-9 proposals. The Program committee will share the exact deadline to submit reviews in Pretalx (typically end of March). Please submit reviews as soon as possible; it will allow us to finalize and announce the schedule sooner!
+
+SciPy has adopted a double-open review process. Reviewers will see the name(s) of the submitters when they review a submission, and their name will be attached to their review. Their name will also be listed on the website as a member of the Program Committee.
+
+Reviewers may be asked to review submissions for talks and tutorials. Tutorials should be approachable to a broad audience in the SciPy community, so all members of the reviewer pool are eligible to review them. Please refer to the tips for tutorial review for what we'd like to see in tutorial reviews. If you feel you are not qualified to review a tutorial that you are assigned, please contact the tutorials chairs.
+
+Instructions
+------------
+
+.. image:: pretalx.jpeg
+  :width: 400
+  :alt: One does not simply Pretalx
+
+Go to https://cfp.scipy.org/orga/event/2024/reviews/ to see your assigned proposals.
+Each proposal will ask you to provide the following:
+
+- **"Overall Evaluation"** (score +2 to -2)
```

Comment:
for the +2 to -2 we usually have a key of what that means. Would it be helpful to add it here too?

Reply:
I don't have the exact language that we used in the Pretalx review form, I suggest to add it later once that form goes online.